client/blocks/credit-card-form/index.jsx

```diff
@@ -208,11 +208,11 @@ function SaveButton( { translate, formSubmitting } ) {
 	return (
 		<FormButton disabled={ formSubmitting } type="submit">
 			{ formSubmitting
-				? translate( 'Saving Card…', {
+				? translate( 'Saving card…', {
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.